index.js

```diff
@@ -1,12 +1,29 @@
-function destroyVue() {
-  this.$destroy();
-  document.removeEventListener('turbolinks:before-cache', destroyVue.bind(this))
+// TL means Turbolinks
+function isTLCacheDisabled() {
+  var meta = document.head.querySelector('[name="turbolinks-cache-control"]');
+
+  return meta && meta.content === 'no-cache';
+}
+
+function getTLTeardownEvent() {
```

Comment:
For readability sake, I think it's ok to name this function `getTurbolinksTeardownEvent()`